chpl/chpl-service/pom.xml

```diff
@@ -433,7 +433,7 @@
                         <source>src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceNonconformityDocument.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceNonconformityStatus.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceRequirement.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceRequirementType.java</source>
+                        <source>src/main/java/gov/healthit/chpl/domain/surveillance/RequirementGroupType.java</source>
```

Comment:
Are these the things that need to be alphabetical? If not, should they be anyway?